docs/filters.md

```diff
@@ -306,6 +306,11 @@ Aliases: `cn`
 
 HubTurbo automatically downloads detailed information about issues when the [`updated`](#updated) filter is specified, and then displays them within the issue cards in the form of a timeline.
 
+HubTurbo also supports `;` as a shorter form of OR that can be used to unify a disjunction of several qualifiers of the same type.
+For example, `(repo:a OR repo:b OR repo:c) AND has:d` can be written as `repo:a;b;c has:d`.
+
+To refer to some specific issues in a repo, use `id:repo#issue_num`. For example, `(repo:repo1 AND id:3) OR (repo:repo2 AND id:3 .. 4)` can be written as `id:repo1#3 OR id:repo2#3 .. 4`
```

Comment:
Could this part go under the id qualifier section? We can add something to its 'expects' text, as well as to the 'formats' section above it, on qualified issue ids.